### ISSUE TITLE
[Issue] Certain Cross world tp takes to the coords in the same dimensions Enigmatica 2 Pack

### ISSUE THREAD

**Issue by N0VVAH** [magmafoundation/Magma#439]
**Describe the issue**
for /home and /spawn I am using FTB Utils mod included in the pack
When using /home or /spawn on the Moon takes to the corresponding coordinates in those Dimensions.
Running /home or /spawn in the nether take you to a portal in the nether. Its hard to tell but it seems like it takes you to a portal in the overworld and instantly teleports back in to the nether. Leaving at a portal in the nether.
/Home and /spawn in the End always takes you back to the world spawn in the overworld dimension
In a similar vein the rocket from advanced rocketry on the moon doesn't take you back to the overworld just goes up to y level 1000 and goes back down

**To Reproduce**
Run Enigmatica 2 server will magma.
set a home in the overworld, that isn't at spawn
Go to nether and run /spawn and /home
go to the moon and run /spawn and/home, or try to ride a rocket out
Go to end and run /home

**Expected behavior**
running /home should take you to the coords set in the right dimension

**Info (please complete the following information):**
- Specific mods that are having issues:
   - FTB Utilities - 5.4.1.131
   - Advanced Rocketry - 1.12.2-1.7.0-223
- Magma Version: 48ec0a1
- OS: Linux

**Additional context**
Mods: Enigmatica 2 Pack latest version
Plugins: 
- Buycraft - lastest
- DeluxeChat - 1.13.2
- LuckPerms - latest
- MVdWPlaceholder - latest
- nuvotifier - latest
- PlaceholderAPI - latest
- Suberbvote - latest
- TAB - latest
- Vault - latest

[latest.log] https://mclo.gs/c2QmOPY

**Comment by UeberallGebannt** [comment]
Can confirm this for FTBU, seems like if you use /home in the nether and the home is in the overworld, in my case I just get teleported to the location where I entered the dimension in the overworld (the portal in this case)

**Comment by N0VVAH** [comment]
Yeah that is the same issue I have with it. Can I ask if you have the same issue with advanced rocketry's moon dimension?
